install/detect-platform.sh

```diff
@@ -19,7 +19,22 @@ if [[ $CONTAINER_ENGINE == "podman" ]]; then
   FORMAT="{{.Host.Arch}}"
 fi
 
-export DOCKER_ARCH=$($CONTAINER_ENGINE info --format "$FORMAT")
+DOCKER_ARCH_OUTPUT=$($CONTAINER_ENGINE info --format "$FORMAT" 2>&1)
+DOCKER_INFO_EXIT_CODE=$?
+
+if [[ $DOCKER_INFO_EXIT_CODE -ne 0 ]]; then
+  echo "FAIL: Unable to get $CONTAINER_ENGINE architecture information."
+  echo "$DOCKER_ARCH_OUTPUT"
+  if [[ "$DOCKER_ARCH_OUTPUT" == *"permission denied"* ]]; then
+    echo ""
+    echo "You may need to add your user to the docker group:"
+    echo "  sudo usermod -aG docker \$USER"
+    echo "Then log out and log back in, or run: newgrp docker"
+  fi
```

Comment:
There are two options from here, actually:
1. Include yourself into the docker group
2. Run the install.sh script with sudo